src/components/Breadcrumbs/Breadcrumbs.styles.ts

```diff
@@ -0,0 +1,48 @@
+import { Link } from 'react-router-dom';
+import { styled } from '@mui/system';
+import { Box } from '@mui/material';
+
+import theme from 'styles/theme';
+import { variables } from 'styles/variables';
+import { StyledFlexAllCenter } from 'styles/styledComponents/Flex';
+
+const commonCenterStyles = `
+  display: flex;
```

Comment:
you can use StyledFlexTopCenter as well

Reply:
no, these styles are used by Link component